purchase_order_line_menu/views/purchase_order_line_views.xml

```diff
@@ -1,20 +1,165 @@
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_id']" position="after">
+                    <field name="state" optional="hide" />
+                    <field name="invoice_status" optional="hide" />
+            </xpath>
+            <xpath expr="//field[@name='product_id']" position="after">
+                    <field name="product_type" optional="hide" />
+            </xpath>
+            <xpath expr="//field[@name='product_qty']" position="after">
+                    <field name="qty_invoiced" optional="hide" />
+                    <field name="qty_to_invoice" optional="hide" />
+                    <field name="qty_received" optional="hide" />
+                    <field name="qty_received_manual" optional="hide" />
+            </xpath>
+            <xpath expr="//field[@name='price_subtotal']" position="after">
+                    <field name="taxes_id" widget="many2many_tags" optional="hide" />
+                    <field
+                    name="price_tax"
+                    widget="monetary"
+                    optional="hide"
+                    sum="Price Tax"
+                />
+                    <field name="price_total" optional="hide" sum="Price Total" />
+            </xpath>
+            <xpath expr="//field[@name='price_subtotal']" position="attributes">
+                <attribute name="sum">Price Subtotal</attribute>
+            </xpath>
+            <xpath expr="//field[@name='date_planned']" position="before">
+                    <field name="date_order" optional="show" widget="date" />
+            </xpath>
+            <xpath expr="//tree" position="inside">
+                    <field name="product_packaging_id" optional="hide" />
+            </xpath>
+            <xpath expr="//field[@name='name']" position="attributes">
+                <attribute name="optional">show</attribute>
+            </xpath>
+            <xpath expr="//field[@name='date_planned']" position="attributes">
+                <attribute name="optional">hide</attribute>
+            </xpath>
+            <xpath expr="/tree" postion="inside">
+                    <field
+                    name="invoice_lines"
+                    widget="many2many_tags"
+                    groups="account.group_account_invoice"
+                    optional="hide"
+                />
+            </xpath>
+        </field>
+    </record>
+
     <record model="ir.ui.view" id="view_purchase_order_line_pivot">
         <field name="name">purchase.order.line.pivot</field>
         <field name="model">purchase.order.line</field>
         <field name="arch" type="xml">
             <pivot string="Order Lines" sample="1">
-                <field name="product_id" type="row" />
-                <field name="product_qty" type="col" />
-                <field name="price_subtotal" type="measure" />
+                    <field name="product_id" type="row" />
+                    <field name="date_order" type="col" />
```

Comment:
It seems to cause this error when trying to switch to the pivot view.
![image](https://github.com/OCA/purchase-workflow/assets/7766939/a3628b99-eeec-46bb-bf8c-a69b0e790088)

Reply:
Fixed.
Thank you @yostashiro